src/index.ts

```diff
@@ -219,8 +225,9 @@ function getInputs(): ActionArgs {
       core.getInput(INPUTS.ANNOTATIONS, { required: false }),
     ) || {}) as KeyVal,
     baseUrl: core.getInput(INPUTS.BASE_URL, { required: true }),
-    clientId: core.getInput(INPUTS.CLIENT_ID, { required: true }),
-    clientSecret: core.getInput(INPUTS.CLIENT_SECRET, { required: true }),
+    clientId: core.getInput(INPUTS.CLIENT_ID, { required: false }),
+    clientSecret: core.getInput(INPUTS.CLIENT_SECRET, { required: false }),
+    accessToken: core.getInput(INPUTS.CLIENT_SECRET, { required: false }),
```

Comment:
Params should be in alphabetical order. If you want to get fancy you can update this to ensure that either clientid/secret is set OR accessToken is set (but not both)